plugin/ale/aaflags.vim

```diff
@@ -40,3 +40,8 @@ endif
 if !exists('g:ale_echo_cursor')
     let g:ale_echo_cursor = 1
 endif
+
+" Set this to the location of the jshint configuration file
+if !exists('g:ale_jshint_config_loc')
+    let g:ale_jshint_config_loc = '.jshintrc'
+endif
```

Comment:
Oh sorry, I only just saw this now. This conflicts with some recent changes I did. Check my comments about reading from parent directories as  default. I would maybe move this flag into the linter file itself, and keep only the options which apply to all linters in this file.